ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -104,7 +104,9 @@ def save_replay_buffer(self) -> None:
         Save the training buffer's update buffer to a pickle file.
         """
         filename = os.path.join(self.artifact_path, "last_replay_buffer.hdf5")
-        logger.info(f"Saving Experience Replay Buffer to {filename}")
+        logger.info(
+            f"Saving Experience Replay Buffer to {filename} ({os.path.getsize(filename)} bytes)"
```

Comment:
nit: Isn't this the old size? Maybe better to print after saving.

Reply:
This shouldn't have been merged - let me revert. It will actually crash since initially there is no `last_replay_buffer.hdf5`

Reply:
Updated to two messages:
```
2021-03-31 14:28:45 INFO [trainer.py:107] Saving Experience Replay Buffer to results/ppo/PushBlock/last_replay_buffer.hdf5...
2021-03-31 14:28:45 INFO [trainer.py:112] Saved Experience Replay Buffer (1591310 bytes).
```
So that users won't be wondering why the program is stuck (if the replay buffer is gigs in size).